ModelRocketTests/JSONTests.swift

```diff
@@ -55,7 +55,8 @@ class JSONTests: XCTestCase {
         
         // No data
         let emptyJSON = JSON(data: nil)
-        XCTAssertTrue(emptyJSON.isNil, "JSON not nil")
+        XCTAssertFalse(emptyJSON.hasKey, "JSON not nil")
```

Comment:
Technically, this message should now be "JSON is not nil" (since hasKey == !isNil)